tests/043_dyn_import_error.ts

```diff
@@ -0,0 +1,9 @@
+async function main(): Promise<void> {
+  const root = `file://${Deno.cwd()}/`;
```

Comment:
This probably doesn't work with windows?

Reply:
CI was green, anyway I removed this integration test